DependencyInjection/Configuration.php

```diff
@@ -73,11 +73,28 @@ public function getConfigTreeBuilder()
                 ->thenInvalid('The useSsl and useStartTls options are mutually exclusive.')
             ->end();
 
+        $this->addManagerSection($rootNode);
+
         $this->addServiceSection($rootNode);
 
         return $treeBuilder;
     }
 
+    private function addManagerSection(ArrayNodeDefinition $node)
+    {
+        $node
+            ->addDefaultsIfNotSet()
+                ->children()
+                    ->arrayNode('manager')
+                        ->addDefaultsIfNotSet()
+                        ->children()
+                            ->scalarNode('user_class')->defaultValue('FR3D\LdapBundle\Model\LdapUser')->end()
```

Comment:
@Noles Please put this setting inside of the user subtree after `arrayNode('attributes')`